src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -544,31 +878,76 @@ ConvAsmImplicitGemmGTCDynamicWrwXdlops::GetSolution(const ConvolutionContext& ct
     result.construction_params.push_back(kernel);
 
     const auto& conv_problem = ctx.conv_problem;
+    const auto& lowp_quant   = ctx.conv_problem.GetConv().lowp_quant;
 
-    result.invoker_factory = [conv_problem,
-                              log2_gemm_k_global_splits](const std::vector<Kernel>& kernels) {
+    auto opShapeArgs = ComputeDynamicIGemmWrwKernelArgs(conv_problem, log2_gemm_k_global_splits);
+
+    result.invoker_factory = [=](const std::vector<Kernel>& kernels) {
         return [=](const Handle& handle, const AnyInvokeParams& primitive_parameters) {
             decltype(auto) data_ctx = primitive_parameters.CastTo<conv::WrWInvokeParams>();
```

Comment:
[Note] data_ctx is used in cases like
```
data_ctx = primitive_parameters.CastTo<conv::DataInvokeParams>();
```
Obtaining "data_something" from "WrWInvokeParams" looks like nonsense ;) Perhaps `invoke_params` is still the best choice...

Reply:
@atamazov I try to use `DataInvokeParams`. But it didn't work. As `DataInvokeParams.tensors` is type `ConvDataTensors`, and its `w` is a `ConstData_t`, I can not write to weight for wrw. But wrw needs to update weights.
Otherwise, we setup an another PR to change `ConvDataTensors` struct?